app/aboot/recovery.c

```diff
@@ -428,8 +428,8 @@ int _emmc_recovery_init(void)
 		boot_into_recovery = 1;
 	}
 
-	if (!strcmp("update-radio",msg->command))
-	{
+#if !ABOOT_STANDALONE
+	if (!strcmp("update-radio",msg->command)) {
```

Comment:
I'd appreciate if we could avoid churn here, even if the old code is slightly dumber. I'd prefer this only add the `#if` and `#endif` and doesn't reformat the code inbetween